packages/analytics/src/index.ts

```diff
@@ -84,8 +85,18 @@ const sendEventCreator = ({ events, key, user }: Config) => (
   emitter.emit(event, properties);
 
   const currentUser = user ? user : getUser();
-
-  return api({ key, event, properties, user: currentUser }, endpoint);
+  const currentContext = getContext();
+
+  return api(
+    {
+      key,
+      event,
+      properties,
+      user: currentUser,
+      context: currentContext,
```

Comment:
Here as well let us not send it if it does not exist

Reply:
Not sent if null